recipes/libflint/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+source activate "${CONDA_DEFAULT_ENV}"
+
+chmod +x configure
+
+echo "int flint_test_multiplier(){return 1;}" > test_helpers.c
```

Comment:
So, I'm guessing this is the test that file that caused this to hang. I'm ok with skipping it and this seems as good of way as any. Though could we please add a comment above explaining what we are doing and why?